examples/fortran/wrapper_test/wrapper_test.f90

```diff
@@ -36,7 +36,7 @@ program main
     real(8), dimension(:), allocatable :: species_x, wdot, species_y
     real(8), dimension(:), allocatable :: mwi
     
-    mixture     = "air_11"
```

Comment:
This should not be changed.  `air_11` is the correct name.

Reply:
Ok. That was a leftover of the rebase. Sorry!